vue-client/src/resources/json/combinedTemplates.json

```diff
@@ -964,6 +964,196 @@
         }
       }
     },
+    "manuscriptMonographText": {
+      "label": "Handskrift",
+      "description": "Handskrift monografi - RDA",
+      "value": {
+        "record": {
+          "@id": "https://id.kb.se/TEMPID",
+          "@type": "Record",
+          "mainEntity": {
+            "@id": "https://id.kb.se/TEMPID#it"
+          },
+          "bibliography": [],
+          "descriptionConventions": [
+            {
+              "@id": "https://id.kb.se/marc/Isbd"
+            },
+            {
+              "@type": "DescriptionConventions",
+              "code": "rda"
```

Comment:
A link would be good (as discussed); but that is reasonably done in a separate PR for all templates.

Reply:
Yes, let's do that separate.